bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/AbstractArrayValue.java

```diff
@@ -131,6 +131,15 @@
     @Deprecated
     public abstract String getString(long index);
 
+    /**
+     * Get string value in the given index.
+     *
+     * @param index array index
+     * @return array element
+     */
+    @Override
+    public abstract BString getBString(long index);
```

Comment:
why did we add this in this PR?

Reply:
Actually no need for this here. I forgot to remove it. I'll clean it and update